Data/DataModel/Response.cs

```diff
@@ -80,7 +81,15 @@ public static string FormatSubmission(string submission)
                 return string.Empty;
             }
 
-            return Regex.Replace(submission, @"[^a-zA-Z\d]", string.Empty).ToUpper();
+            // Use Compatibility Decomposition (FormKD) so that
+            // 1. Compatible equivalents like exponents, ligatures, and Roman numerals are converted into preserved common characters.
+            // 2. Canonical composite characters like the accented e (\u00E9) characters in re'sume' are decomposed to preserve the starter characters.
+            if (!submission.IsNormalized(NormalizationForm.FormKD))
+            {
+                submission = submission.Normalize(NormalizationForm.FormKD);
+            }
+
+            return Regex.Replace(submission, @"[^a-zA-Z\d]", string.Empty).ToUpperInvariant();

```

Comment:
What is the difference between the current ToUpper and ToUpperInvariant?

Reply:
The main difference is that `ToUpper` uses the thread's _current_ locale while `ToUpperInvariant` uses the invariant locale. The most common place where they can get different results is if the `Thread.CurrentThread.CurrentCulture` is Turkish (Turkey) where i->İ, as opposed to most of the rest of the world where i->I. (note the dot over the Turkish version since dotted and dotless 'I' are distinct characters in the Turkish alphabet).

Reply:
Awesome - thanks for the explanation :)